Makefile

```diff
@@ -0,0 +1,14 @@
+PWD := $(shell pwd)
```

Comment:
This also should've been at least in a separate commit

Reply:
:ok_hand: